uploadNeuroDB/tarchiveLoader

```diff
@@ -484,6 +484,24 @@ if ($mcount < 1) {
     exit $NeuroDB::ExitCodes::NO_VALID_MINC_CREATED;
 }
 
+################################################################
+################################################################
+# Make sure the mri_protocol table no longer has comma-separated
+########################## values ##############################
+################################################################
+my ($isMRIProtocolValid) = $utility->isValidMRIProtocol();
+
+if ( !($isMRIProtocolValid) ) {
+    my $message = "\nComma separated ranges or values in the mri_protocol "
+                  . "table are no longer supported. Please modify your "
+                  . "mri_protocol table accordingly. Exiting now. \n";
+    $notifier->spool('tarchive loader', $message, 0,
+		    'tarchiveLoader', $upload_id, 'Y',
+		    $notify_notsummary);
+    print STDERR $message;
+    exit $NeuroDB::PROJECT_CUSTOMIZATION_FAILURE;
```

Comment:
oups. I missed that earlier and noticed it when testing the code. It should be `$NeuroDB::ExitCodes::PROJECT_CUSTOMIZATION_FAILURE;` and not `$NeuroDB::PROJECT_CUSTOMIZATION_FAILURE;`

Reply:
yep, my bad again!